packages/editor-ui/src/views/SettingsApiView.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+	<SettingsView>
+		<div :class="$style.container">
+			<div :class="$style.header">
+				<n8n-heading size="2xlarge">
+					{{ $locale.baseText('settings.api') }}
+				</n8n-heading>
+			</div>
+
+			<div v-if="apiKey">
+				<p class="mb-s">
+					<n8n-text color="text-base">
+						<font-awesome-icon icon="info-circle" />
+						{{ $locale.baseText('settings.api.view.info') }}
+						<n8n-link to="https://docs.n8n.io/api/">
+							{{ $locale.baseText('generic.learnMore') }}
+						</n8n-link>
+					</n8n-text>
+				</p>
+				<n8n-card :class="$style.card">
+					<span :class="$style.delete">
+						<n8n-link @click="showDeleteModal">
+							{{ $locale.baseText('generic.delete') }}
+						</n8n-link>
+					</span>
+					<CopyInput
+						:label="$locale.baseText('settings.api.view.myKey')"
+						:value="apiKey"
+						:toast-title="$locale.baseText('settings.api.view.copy.toast')"
+					/>
+				</n8n-card>
+			</div>
+			<div :class="$style.placeholder" v-else>
+				<n8n-heading size="xlarge">
+					{{ $locale.baseText('settings.api.create.title') }}
+				</n8n-heading>
+				<p class="mt-2xs mb-l">
+					<n8n-text color="text-base">
+						{{$locale.baseText('settings.api.create.description')}}
+						<n8n-link to="https://docs.n8n.io/api/">
+							{{$locale.baseText('settings.api.create.description.link')}}
+						</n8n-link>
+					</n8n-text>
+				</p>
+				<n8n-button :loading="loading" size="large" class="mt-l" @click="createApiKey">
+					{{$locale.baseText(loading ? 'settings.api.create.button.loading' : 'settings.api.create.button')}}
+				</n8n-button>
+			</div>
+		</div>
+	</SettingsView>
+</template>
+
+<script lang="ts">
+import {createApiKey, deleteApiKey, getApiKey } from '../api/api-keys';
+import { showMessage } from '@/components/mixins/showMessage';
+import { IUser } from '@/Interface';
+import mixins from 'vue-typed-mixins';
+
+import SettingsView from './SettingsView.vue';
+import CopyInput from '../components/CopyInput.vue';
+import {DELETE_API_KEY_MODAL_KEY} from "../constants";
+
+export default mixins(
+	showMessage,
+).extend({
+	name: 'SettingsPersonalView',
+	components: {
+		SettingsView,
+		CopyInput,
+	},
+	data() {
+		return {
+			loading: false,
+			mounted: false,
+			error: '',
+		};
+	},
+	mounted() {
+		this.getApiKey();
+	},
+	computed: {
+		currentUser() {
+			return this.$store.getters['users/currentUser'] as IUser;
+		},
+		apiKey() {
+			return this.$store.getters['settings/apiKey'];
+		},
```

Comment:
nitpick - would add return types everywhere in computed so that type checks take effect when in VS Code with Vueter

Reply:
Good point. Fixed!